joint_state_broadcaster/src/joint_state_broadcaster_parameters.yaml

```
@@ -30,24 +34,35 @@ joint_state_broadcaster:
     position: {
       type: string,
       default_value: "position",
+      read_only: true,
     }
     velocity: {
       type: string,
       default_value: "velocity",
+      read_only: true,
     }
     effort: {
       type: string,
       default_value: "effort",
+      read_only: true,
     }
   use_urdf_to_filter: {
     type: bool,
     default_value: true,
+    read_only: true,
     description: "Uses the robot_description to filter the ``joint_states`` topic.
     If true, the broadcaster will publish the data of the joints present in the URDF alone.
     If false, the broadcaster will publish the data of any interface that has type ``position``, ``velocity``, or ``effort``."
   }
   frame_id: {
     type: string,
     default_value: "base_link",
+    read_only: true,
     description: "The frame_id to be used in the published joint states. This parameter allows rviz2 to visualize the effort of the joints."
   }
+  publish_dynamic_joint_states: {
+    type: bool,
+    default_value: true,
```

```suggestion
    default_value: false,
```

We can change this in the backports

---

Whatever you say boss 🫡